content/en/pr_gates/search/_index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Search and Manage PR Gate Rules 
+description: Learn how to search for your PR Gate rules.
+aliases:
+  - /quality_gates/search/
+further_reading:
+- link: "/pr_gates"
+  tag: "Documentation"
+  text: "Learn about PR Gates"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">PR Gates is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+## Overview
+
+The [**PR Gate Rules** page][1] is useful for developers who want to keep an eye on the PR gates for their build pipelines. You can see all of the rules defined by the organization.
+
+{{< img src="pr_gates/rules_list_2.png" alt="PR Gate rules in Datadog displaying the rule name, evaluation, scope, blocking status, last modified date, and the creator avatar" style="width:100%" >}}
+
+Use this page to create a rule, edit an existing rule, or click on a rule to investigate its past executions.
+
+## Search for rules
+
+To see your PR gate rules, navigate to [**Software Delivery** > **PR Gates** > **PR Gate Rules**][1].
```

Comment:
```suggestion
To see your PR Gate Rules, navigate to [**Software Delivery** > **PR Gates** > **PR Gate Rules**][1].
```

Reply:
I don't think we've been capitalizing "rule" in this context, unless it's a title or UI label.
```suggestion
To see your PR Gate rules, navigate to [**Software Delivery** > **PR Gates** > **PR Gate Rules**][1].
```